omod/.gitignore

```diff
@@ -0,0 +1,68 @@
+# Copied from https://github.com/github/gitignore
```

Comment:
Please remove the git ignore file. You do not need to commit the file

Reply:
How are you setting the feature to be only available for KNH and not any other facility? Are there some components missing in this pr?